### ISSUE TITLE
update resource_aws_route53_health_check to support Enabling/Disabling Healthcheck

### ISSUE THREAD

**Issue by Giri-Chintala** [terraform-providers/terraform-provider-aws#10117]
AWS HealthCheck API supports Enabling/Disabling Healthcheck. https://docs.aws.amazon.com/Route53/latest/APIReference/API_UpdateHealthCheck.html#API_UpdateHealthCheck_RequestSyntax#Disabled

Currently, terraform module resource_aws_route53_health_check is not handling disabled state information. Request is to update terraform to include HealthCheck disabled state information. 

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

I'm planning to work on this enhancement. 

### Description

HealthCheck Disabled state information is helpful for finding/manipulating Health status of Route53 HealthCheck.

### New or Affected Resource(s)

* aws_route53_health_check

### Potential Terraform Configuration

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

### References

https://docs.aws.amazon.com/Route53/latest/APIReference/API_UpdateHealthCheck.html#API_UpdateHealthCheck_RequestSyntax#Disabled

* #0000

**Comment by artburkart** [comment]
This is a duplicate of https://github.com/terraform-providers/terraform-provider-aws/issues/8937

**Comment by anGie44** [comment]
This feature has been merged and will release with `v3.2.0` of the Terraform AWS Provider, expected out later today.
